src/FixedDataTable.js

```diff
@@ -1026,25 +1026,12 @@ var FixedDataTable = createReactClass({
         );
 
         var previousColumnsWidth = 0;
-        for (i = 0; i < scrollableColumnIndex; ++i) {
+        for (i = 0; i < scrollableColumnIndex - 1; ++i) {
           column = columnInfo.bodyScrollableColumns[i];
           previousColumnsWidth += column.props.width;
         }
 
-        var availableScrollWidth = adjustedWidth - totalFixedColumnsWidth;
-        var selectedColumnWidth = columnInfo.bodyScrollableColumns[
-          scrollableColumnIndex
-        ].props.width;
-        var minAcceptableScrollPosition =
-          previousColumnsWidth + selectedColumnWidth - availableScrollWidth;
```

Comment:
But previousColumnsWidth only sums up the scrollable columns, so we don't need to account for fixed width? And I don't follow what you mean by 'make it the first column' can you elaborate? My read of this is that to get to the third (scrollable) column, we need to offset the scroll by the distance of the first 2 (scrollable) columns. Here is a video demonstrating what happens if I add scrollToColumn(4) on the reorder example : 
![scrolltocolumn](https://user-images.githubusercontent.com/294747/27243038-4705ab52-529d-11e7-8434-d574fc2d0fd4.gif)